internal/wasm/interpreter/interpreter.go

```diff
@@ -425,16 +424,22 @@ func (it *interpreter) lowerIROps(f *wasm.FunctionInstance,
 }
 
 // Call implements an interpreted wasm.Engine.
-func (it *interpreter) Call(ctx *wasm.ModuleContext, f *wasm.FunctionInstance, params ...uint64) (results []uint64, err error) {
-	prevFrameLen := len(it.frames)
+func (e *engine) Call(ctx *wasm.ModuleContext, f *wasm.FunctionInstance, params ...uint64) (results []uint64, err error) {
+	paramSignature := f.FunctionType.Type.Params
```

Comment:
this is the only new code. everything else is due to renaming interpreter to engine